### ISSUE TITLE
Add support for text fields in ms word

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8048]
### Steps to reproduce:
1. Open MS Word
2. Create two text fields

### Expected behavior:
NVDA should Report "textfield 1" or similar, when the caret is moving into the textfield Region. Pressing enter on the textfield should simulate a mouse click so that the caret jumps into the text field. Escape should leave the text field.

### Actual behavior:
NVDA does not Report there are text fields in a document. It reports only "blank". NVDA does not treat text fields as individual objects, so they cannot be Accessed by the user unles the user performs a mouse click on the text field.

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Both

Other information:

Windows version:
Windows 8.1

Name and version of other software in use when reproducing the issue:
MS Word 2010

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
no

**Comment by PratikP1** [comment]
I believe that additional support needs to be considered for modern form fields in Word. Forms created with modern fields do not generally behave as they should.

**Comment by leonardder** [comment]
This looks like a duplicate of #6944.

**Comment by Adriani90** [comment]
Closing in favor of #6944.
